routes/rename-project.js

```diff
@@ -0,0 +1,25 @@
+var utils = require("./utils");
+
+module.exports = function(config) {
+  return function(req, res) {
+    var project = JSON.parse(JSON.stringify(req.session.project.meta));
```

Comment:
I don't understand what you're doing here. It looks like you convert an object to a string, and then back to an object.